example/src/demos/Lines.tsx

```diff
@@ -80,7 +80,7 @@ function EndPoint({ position, onDrag, onEnd }: any) {
   return (
     <mesh position={position} {...bindDrag} {...bindHover}>
       <sphereGeometry args={[7.5, 16, 16]} />
-      <meshBasicMaterial color={hovered ? 'hotpink' : 'white'} />
+      <meshBasicMaterial color={hovered ? 'hotpink' : (([0.8, 0.2, 0.9] as unknown) as typeof THREE.Color)} />
```

Comment:
i think this should be sorted to accept an array, having to typecast as `unknown` first is annoying imo